docs/user-guide/downward-api/volume/dapi-volume.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
```

Comment:
Why do we have 2 yaml files that are fundamentally similar?  Can the example and the top-level doc reference the same YAML?